AEPCampaignClassic/Sources/Extensions/String+256sha.swift

```diff
@@ -0,0 +1,23 @@
+/*
+ Copyright 2022 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
```

Comment:
nit: Suggest renaming files to String+Sha256.swift and Data+Sha256.swift

Reply:
For testing, I think writing tests for just String+256SHA should be okay.. since it completely covers Data+256Sha file